### PR TITLE
Makes bloods splats only spawn if If there is not an existing blood splats

### PR DESCRIPTION
**Pull request by Bod9001**
### what this do?
Makes bloods splats only spawn if If there is not an existing blood splats
### why you do this?
Too many spawn on a dead body
### what you tested?
Editor

### CODE REVIEW

**Comment by daalta** [comment]
I think ideally blood would spread out over the floor instead of vanishing...

But this is still better than the current situation of blood getting stacked to the ceiling!! 😁

**Review by daalta — COMMENTED** [review]
Looks good, testing it in the editor now.

**Comment by Bod9001** [comment]
yeah, it's meant to be a temporary solution until they are probably imprinted into tile effects, instead of individual game objects

File: UnityProject/Assets/Scripts/Factories/EffectsFactory.cs [hunk 1]
```diff
@@ -96,8 +96,12 @@ public static void BloodSplat(Vector3 worldPos, BloodSplatSize splatSize, BloodS
 
 		if (chosenTile != null)
 		{
-			Spawn.ServerPrefab(chosenTile, worldPos,
-				MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true).Objects);
+			var matrix = MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true).Matrix;
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/Factories/EffectsFactory.cs]: don't call AtPoint twice, just cache `MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true)` here once:
```suggestion
			var matrixInfo = MatrixManager.AtPoint(Vector3Int.RoundToInt(worldPos), true);
```

**Comment by daalta** [comment]
Here's what it looks like now. After implementing foma's solution I think we can merge this, though we'll need to add more blood in the future :)

Unity_2020-02-06_22-23-07

**Comment by fomalsd** [comment]
blood should be moved to tile effects anyway in order to be cheaper

**Pull request merged**